pkgs/servers/xteve/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```

Comment:
... and join the rest in the next PR, with the title:

`xteve: init at 2.2.0.200`